### ISSUE TITLE
[MAS4.1.2] Name property is not properly defined for Audio/Video clips.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1314]
**Actual Result:**
When focus lands on the audio clip then narrator is reading 'audio' and when focus lands on the video clip then narrator is reading 'microsoft..', Narrator is not reading proper name for audio/video clips.
Name property does not properly define for Audio/Video clips.
Name of Audio Clip section is not properly defined. So, Screen reader is reading 'Audio' twice for Audio clip present on the page.

**Note:** Same issue repro at All video clips defined on the pages. Attachment - **"MAS4.1.2_Name for video clip is not descriptive"**.

**Expected Result:**
Name property should be properly defined for Audio/Video clips.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/ ) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'accessibility' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Page will upload.
5. Open Narrator (Ctrl+Win.key+Enter).
6. Navigate to Audio/Video clips by using Tab key.
7. Observe that Audio/Video clips name is not defined properly.

**User Impact:**
If Name,State and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[MAS4.1.2_Name is not define for audio and video clips.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2547681/MAS4.1.2_Name.is.not.define.for.audio.and.video.clips.zip)
[MAS4.1.2_Name for video clip isn not descriptive.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2588567/MAS4.1.2_Name.for.video.clip.isn.not.descriptive.zip)

**Comment by corinagum** [comment]
Confirmed as external issue by accessibility team

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
